tests/concat.rs

```diff
@@ -0,0 +1,148 @@
+        assert!(
+            !db1_metadata.concat_competibility(&input_file_list),
+            "Sketches should not be compatible"
+        );
+    }
+
+    #[test]
+    fn test_concat_sketches() {
+        let sandbox = TestSetup::setup();
+
+        Command::new(cargo_bin("sketchlib"))
+            .current_dir(sandbox.get_wd())
+            .arg("sketch")
+            .args(&["--k-vals", "17"])
+            .arg("--seq-files")
+            .arg(sandbox.file_string("14412_3#82.contigs_velvet.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("14412_3#84.contigs_velvet.fa.gz", TestDir::Input))
+            .arg("-v")
+            .args(&["-o", "part1"])
+            .assert()
+            .success();
+        log::info!("Part 1 Sketched");
+
+        Command::new(cargo_bin("sketchlib"))
+            .current_dir(sandbox.get_wd())
+            .arg("sketch")
+            .args(&["--k-vals", "17"])
+            .arg("--seq-files")
+            .arg(sandbox.file_string("R6.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("TIGR4.fa.gz", TestDir::Input))
+            .arg("-v")
+            .args(&["-o", "part2"])
+            .assert()
+            .success();
+        log::info!("Part 2 Sketched");
+
+        Command::new(cargo_bin("sketchlib"))
+            .current_dir(sandbox.get_wd())
+            .arg("sketch")
+            .args(&["--k-vals", "17"])
+            .arg("--seq-files")
+            .arg(sandbox.file_string("R6.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("TIGR4.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("14412_3#82.contigs_velvet.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("14412_3#84.contigs_velvet.fa.gz", TestDir::Input))
+            .arg("-v")
+            .args(&["-o", "concat_ref"])
+            .assert()
+            .success();
+        log::info!("concat_ref Sketched");
+
+        // Overlapping labels fails
+        Command::new(cargo_bin("sketchlib"))
+            .current_dir(sandbox.get_wd())
+            .arg("concat")
+            .arg("part1")
+            .arg("--seq-files")
+            .arg(sandbox.file_string("14412_3#82.contigs_velvet.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("14412_3#84.contigs_velvet.fa.gz", TestDir::Input))
+            .arg("-v")
+            .args(&["-o", "concat_test"])
+            .assert()
+            .failure();
+
+        Command::new(cargo_bin("sketchlib"))
+            .current_dir(sandbox.get_wd())
+            .arg("concat")
+            .arg("part1")
+            .arg("--seq-files")
+            // .arg(sandbox.file_string("fasta_part2.txt", TestDir::Input))
+            .arg(sandbox.file_string("R6.fa.gz", TestDir::Input))
+            .arg(sandbox.file_string("TIGR4.fa.gz", TestDir::Input))
+            .arg("-v")
+            .args(&["-o", "concat_test"])
+            .assert()
+            .success();
+        log::info!("concat_test Sketched");
```

Comment:
Not sure logging does anything in the tests? (a few of these can be removed if so)